src/WrapCalls.cpp

```diff
@@ -53,8 +53,8 @@ void insert_func_wrapper_helper(map<FunctionPtr, SubstitutionMap> &func_wrappers
 void validate_custom_wrapper(Function in_func, Function wrapped, Function wrapper) {
     map<string, Function> callees = find_direct_calls(in_func);
     user_assert(callees.count(wrapper.name()))
-        << wrapped.name() << ".in(" << in_func.name() << ") was called but \""
-        << wrapped.name() << "\" is never used in \"" << in_func.name() << "\"\n";
+        << "Wrapper for \"" << wrapped.name() << "\" in \"" << in_func.name()
+        << "\" was created but is never used inside \"" << in_func.name() << "\"\n";
```

Comment:
This makes it sound like the wrapper is never used, where really the problem is that the original function is never used. I suggest:

"Cannot wrap " << wrapped.name() << " in " << in_func.name() << " because " << in_func.name() << " does not call " << wrapped.name() << "\n";